src/e3/anod/spec.py

```diff
@@ -315,7 +316,10 @@ def build_space_name(self) -> str:
         :rtype: str | None
         """
         if self.enable_name_generator:
```

Comment:
You can rewrite this if like this: `return self.qualifiers_manager.build_space_name if self.enable_name_generator and self.kind != "source" else self.name`

Reply:
Done